test/unit/components/Personalization/actions/setStyle.spec.js

```diff
@@ -0,0 +1,50 @@
+import {
+  selectNodes,
+  removeNode,
+  appendNode,
+  createNode
+} from "../../../../../src/utils/dom";
+import createSetStyle from "../../../../../src/components/Personalization/actions/setStyle";
+
+const cleanUp = () => {
+  selectNodes("div#setStyle").forEach(removeNode);
+  selectNodes("style").forEach(node => {
+    if (node.textContent.indexOf("setStyle") !== -1) {
+      removeNode(node);
+    }
+  });
+};
+
+describe("Presonalization::actions::setStyle", () => {
+  beforeEach(() => {
+    cleanUp();
+  });
+
+  afterEach(() => {
+    cleanUp();
+  });
+
+  it("should set personalized content", () => {
+    const collect = jasmine.createSpy();
+    const setStyle = createSetStyle(collect);
+    const element = createNode("div", { id: "setStyle" });
+    const elements = [element];
+
+    appendNode(document.body, element);
+
+    const settings = {
+      content: { "font-size": "33px", priority: "important" },
```

Comment:
I noticed that `priority` applies to all styles instead of individual styles, which seems like it might be a bit restrictive. It's also a little odd that `priority` is stored on the same level of the object as the styles. Have you considered something like:

```
content: {
  styles: [
    {
      name: "font-size",
      value: "33px",
      priority: "important"
    }
  ]
}
```

It's more verbose but more flexible and more appropriately structured in my opinion.

Reply:
@Aaronius the way Target VEC stores styles is `key` and `value` or `key`, `value` and `priority`. If we want to go the route of having an array of styles, then I'll have to adjust the edge code, so it returns the data in the proposed format.

Reply:
@Aaronius because of conflicts I had to open a new PR: https://github.com/adobe/alloy/pull/108. Please let me know if I should change the format or the current approach is OK.

Reply:
As long as you feel like you understand what I'm trying to communicate and have taken it into consideration, I'm good with whatever you choose. 👍